client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-connected-dialog.js

```diff
@@ -111,22 +112,31 @@ class PaymentMethodStripeConnectedDialog extends Component {
 	};
 
 	getButtons = () => {
-		const { onCancel, onDone, stripeConnectAccount, translate } = this.props;
+		const { onCancel, onDone, isDisconnecting, stripeConnectAccount, translate } = this.props;
 
 		const buttons = [];
 
+		const disabled = isDisconnecting;
+
 		if ( stripeConnectAccount.isActivated ) {
-			buttons.push( { action: 'cancel', label: translate( 'Cancel' ), onClick: onCancel } );
+			buttons.push( {
+				action: 'cancel',
+				disabled: disabled,
```

Comment:
Optional: You can just pass `disabled,` here and in the other objects.

Reply:
Agreed - fixed in https://github.com/Automattic/wp-calypso/pull/19244/commits/ee1e0fee72ef06b3b54d1b39dab0661a7bbf4f54